### DISCUSSION TITLE
clipping temperature dataset with rioxarray package gives '-9223372036854775808' as variable value for clipped lat long coords.

### DISCUSSION THREAD

**Discussion by angana16** [pydata/xarray#4735] [Q&A]
I have ERA5 dataset and calculated a percentile index for gridded lat long data all across India. However, when I use the following codes to clip my dataset with shapefiles of certain cities and its neighbouring areas, I am getting erroneous values for some regions. And the exact same code is giving proper values for other regions.
Please help.
Here is my code snippet:
                     
                               india.rio.set_spatial_dims(x_dim="longitude", y_dim="latitude", inplace=False)
                               india.rio.write_crs("epsg:4326", inplace=False)

                                buffer_aroundDelhi = gpd.read_file('/content/del_buff.shp', crs="epsg:4326", masked=True, chunks=True)
                                ddclip = indtn10.rio.clip(buffer_aroundDelhi.geometry.apply(mapping), buffer_aroundDelhi.crs, drop=True)

**Comment by keewis** [comment]
this seems like a question about `rioxarray` rather than `xarray`. You will probably get a faster reply by asking the [`rioxarray` devs](https://github.com/corteva/rioxarray/discussions) directly.

cc @snowman2

**Comment by snowman2** [comment]
As @keewis  mentioned, please forward this to: https://github.com/corteva/rioxarray/discussions. If you are able to provide input data, that would be helpful.
